1. Auth Flow/app.js

```diff
@@ -18,12 +18,12 @@ var index = require('./routes/index');
 var users = require('./routes/users');
 
 //  acr_values: 'onelogin:nist:level:1:re-auth'
-const baseUri = `https://${ process.env.SUBDOMAIN }.onelogin.com/oidc/2`
+const baseUri = `${ process.env.OIDC_BASE_URI }/oidc/2`
 
 // Configure the OpenId Connect Strategy
 // with credentials obtained from OneLogin
 passport.use(new OneLoginStrategy({
-  issuer: baseUri,
+  issuer: baseUri, 
```

Comment:
Please remove the trailing whitespace. If you're using VS Code this is how you can enable that to happen automatically. https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut